src/main/java/com/umbr3114/controllers/PostController.java

```diff
@@ -50,6 +50,10 @@
         if (title == null || bodyText == null || authorId == null || dropId == null || author == null) {
             halt(HttpStatus.BAD_REQUEST_400, new GeneralResponse(HttpStatus.OK_200, "invalid request").toJSON());
         }
+        postModel = new PostModel();
+        postModel.setAuthorId(authorId);
+        postModel.setBodyText(bodyText);
+        postModel.setTitle(title);
```

Comment:
lines 53-56 were added for some reason and this might be an accident from cleaning up a merge conflict. just a heads up.